Makefile

```diff
@@ -5,17 +5,12 @@ DOCKER_REGISTRY_ENGINEERING =
 DOCKER_REGISTRY_REDHAT =
 ADDITIONAL_ARGUMENTS =
 
-CE_DOCKER = $(shell docker version | grep Version | head -n 1 | grep -e "-ce")
-ifneq ($(CE_DOCKER),)
 DOCKER_REGISTRY_ENGINEERING = docker-registry.engineering.redhat.com
 DOCKER_REGISTRY_REDHAT = registry.access.redhat.com/
 DEV_IMAGE_FULL_NAME = $(DOCKER_REGISTRY_ENGINEERING)/$(DEV_IMAGE_ORG)/$(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
 IMAGE_FULL_NAME = $(DOCKER_REGISTRY_ENGINEERING)/$(DEV_IMAGE_ORG)/$(IMAGE_NAME):$(DEV_IMAGE_TAG)
 CONCREATE_CMD = concreate build --overrides=overrides.yaml --target target-docker --tag $(DEV_IMAGE_FULL_NAME)
-else
 DEV_IMAGE_FULL_NAME = $(DEV_IMAGE_ORG)/$(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
```

Comment:
This is a duplicate declaration.

Reply:
Fixed

Reply:
Sorry, @slaskawi . Did you want to keep the declaration that includes DOCKER_REGISTRY_ENGINEERING ? I thought the original idea of this PR was to have the full name (including registry).